src/main/java/io/anserini/collection/C4Collection.java

```diff
@@ -141,15 +141,18 @@ public static class Document implements SourceDocument {
     protected String id;
     protected String url;
     private String contents;
+    private String contentsURL;
     private String raw;
     private long timestamp;
 
     public Document(JsonNode json, String filename, int jsonLoc) {
       this.raw = json.toPrettyString();
-      this.contents = json.get("text").asText();
 
       this.id = String.format("c4-%s-%06d", filename, jsonLoc);
       this.url = json.get("url").asText();
+      this.contents = json.get("text").asText();
+
+      this.contentsURL =  this.contents + "" + this.url;
```

Comment:
Can you explain what this is for? I don't understand why you're concatenating contents and URL...

Reply:
In trec-misinfo-21, we found incorporating URLs into contents can improve performance. I've talked to @ronakice about this issue before; theoretically, I should've opened another PR.

Reply:
yup, let's move this from here.

Reply:
@ronakice sorry, what do you mean?

Reply:
I've restored the file. Will open up a new PR if needed.